jetty-websocket/websocket-jetty-api/src/main/java/org/eclipse/jetty/websocket/api/annotations/OnWebSocketMessage.java

```diff
@@ -35,9 +35,8 @@
  * <li><code>public void methodName(Reader reader)</code></li>
```

Comment:
Use `{@code ...}` instead of `<code>`.

Reply:
I can do this but I will need to remove the linking to the session with `{@link Session}`.